internal/cephfs/volume.go

```diff
@@ -85,9 +83,19 @@ func getVolumeRootPathCeph(ctx context.Context, volOptions *volumeOptions, cr *u
 	return strings.TrimSuffix(string(stdout), "\n"), nil
 }
 
+type additionalClusterInfo struct {
+	resizeSupported       bool
+	subVolumeGroupCreated bool
+}
+
 func createVolume(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID, bytesQuota int64) error {
-	//TODO: When we support multiple fs, need to hande subvolume group create for all fs's
-	if !cephfsInit {
+	// verify if corresponding ClusterID key is present in the map,
+	// and if not, initialize with default values(false).
+	if _, keyPresent := clusterAdditionalInfo[volOptions.ClusterID]; !keyPresent {
+		clusterAdditionalInfo[volOptions.ClusterID] = &additionalClusterInfo{}
+	}
+
+	if created := clusterAdditionalInfo[volOptions.ClusterID].subVolumeGroupCreated; !created {
```

Comment:
no need to assign to `created`, just use `!clusterAdditionalInfo[volOptions.ClusterID].subVolumeGroupCreated` directly

Reply:
Thanks for the review, I'll make the required changes.